bigtable-hbase/pom.xml

```diff
@@ -41,19 +41,12 @@ limitations under the License.
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId>
             <version>${netty.version}</version>
+            <scope>test</scope>
```

Comment:
There is netty code referenced from non-test code. I don't think test is the correct scope.